app/components-react/windows/go-live/useGoLiveSettings.ts

```diff
@@ -46,6 +47,11 @@ export class GoLiveSettingsModule extends StreamInfoView<IGoLiveSettingsState> {
    */
   async prepopulate() {
     await Services.StreamingService.actions.return.prepopulateInfo();
+
+    // TODO: investigate why mutations from `StreamingService.prepopulateInfo` are not applied at this moment
```

Comment:
I think we discovered this issue before - I had a mutation refactor that was pretty ambitious and it caused some issues.  We never revisited it.

Reply:
I tried to understand what is going on here. Yeah, it seems we have an issue with mutations order when we have a deal with Promises. That requires some time to investigate the issue and it's going to be a very dangerous change. So I prefer to revisit it at the moment when I'm going to migrate all our services to Redux.  So I decided to stay with this dirty `sleep` fix here

Reply:
Let's hold off on this PR until we fix it.  I can look into it, since I already tried fixing it before.  And yeah it was a dangerous change that we ended up having to revert 😬

Reply:
https://github.com/stream-labs/streamlabs-obs/pull/2757 I actually made this fix last year which was smaller and didn't get reverted.  It was meant to fix this issue, but maybe it didn't catch all the cases.